packages/lesmis/src/views/ExportView.js

```diff
@@ -87,18 +89,34 @@ const getChildren = ({
   );
 };
 
-export const ExportView = ({ viewProps, viewType, className }) => {
-  const { getExtraExportViewProps, PageContainer, PageContent } = EXPORT_VIEWS[viewType];
-  const exportViewProps = { ...viewProps, ...getExtraExportViewProps() };
-
+const getExportableDashboardsForDownloadView = dashboardValue => {
   const { dropdownOptions } = useDashboardDropdownOptions();
-  const profileDropDownOptions = dropdownOptions.filter(({ exportToPDF }) => exportToPDF);
+  const profileDropDownOptions = dropdownOptions.filter(({ value }) => value === dashboardValue);
   const { exportableDashboards } = getExportableDashboards(profileDropDownOptions);
+  return exportableDashboards;
+};
+
+export const ExportView = ({
+  viewProps,
+  viewType,
+  className,
+  previewDashboards,
+  previewDashboardValue,
+}) => {
+  const { getExtraExportViewProps, PageContainer, PageContent } = EXPORT_VIEWS[viewType];
+  const exportViewProps = { ...viewProps, ...getExtraExportViewProps() };
+  const exportableDashboards =
+    viewType === DASHBOARD_EXPORT_PREVIEW
+      ? previewDashboards
+      : getExportableDashboardsForDownloadView(exportViewProps.exportOptions.dashboard);
```

Comment:
Can we get `dashboardValue` directly from URL, then no need to pass in `previewDashboards` and `previewDashboardValue`? If so it helps code reading. 

`previewDashboards` should be the same as `exportableDashboards`? It can be retrieved by `getExportableDashboardsForDownloadView()`.

Reply:
Yes, great idea. Have reduced the props now.